addons/account/models/partner.py

```diff
@@ -285,6 +285,11 @@ def _journal_item_count(self):
             partner.journal_item_count = self.env['account.move.line'].search_count([('partner_id', '=', partner.id)])
             partner.contracts_count = self.env['account.analytic.account'].search_count([('partner_id', '=', partner.id)])
 
+    @api.multi
```

Comment:
for stat button fields try whenever possible to be multi and to do the job in O(1) sql queries, using count or group by.